docs/CONFIG.md

```diff
@@ -125,6 +143,25 @@ which allow for configuration via the `TMPDIR`, `TEMP` or `TMP` env
 variables, and generally default to
 [something reasonable per your OS](https://docs.python.org/3/library/tempfile.html#tempfile.gettempdir).
 
+## GCP default project
+
+To specify the default project to use in GCP operations (currently
+this includes interfacing with GCS buckets), you can either set the
+environment variable `GCP_PROJECT` to your project name or configure a
+INI-style file in one of the following locations:
+
+* `/etc/bluelabs/records_mover/app.ini`
+* `/etc/xdg/bluelabs/records_mover/app.ini`
+* `$HOME/.config/bluelabs/records_mover/app.ini`
+* `./.bluelabs/records_mover/app.ini`
+
+Example file:
+
+```ini
+[gcp]
+default_project = "my_gcp_project_name"
+```
```

Comment:
The global config file is the approach we'll go with internally to point to our most commonly used scratch bucket and default project.